### ISSUE TITLE
master-SNAPSHOT is not available

### ISSUE THREAD

**Issue by georgeroman** [hyperledger/iroha-java#63]
This is what I get when compiling a project that depends on `master-SNAPSHOT` of `iroha-java`:
```
Could not find artifact com.github.hyperledger:iroha-java:jar:master-219d32262a-1 in jitpack.io (https://jitpack.io)
```

I suspect this is the result of the failed build of the last commit (https://github.com/hyperledger/iroha-java/commit/219d32262aaf8c3ce552023e2422e5c409661633).

**Comment by Alexey-N-Chernyshov** [comment]
Thanks, @georgeroman!

We found an issue in the CI, fix will be soon.

You can use [Telegram](https://t.me/hyperledgeriroha) or [Hyperledger RocketChat](https://chat.hyperledger.org/channel/iroha) for quick response on your issues.

**Comment by Alexey-N-Chernyshov** [comment]
Fixed in #65.
